test/parallel/test-whatwg-encoding-fatal-streaming.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+// From: https://github.com/w3c/web-platform-tests/blob/master/encoding/textdecoder-fatal-streaming.html
```

Comment:
Should these be like https://github.com/nodejs/node/blob/c5f243a5f5417af24c10b294d1d57d5dbf64fea3/test/parallel/test-whatwg-url-searchparams-append.js#L8-L13?

Reply:
No, the URL tests use a completely different format. These had to be ported over to work with out test suite.

Reply:
Can you put the link to a specific sha instead of `blob/master`? Otherwise we lose track of the version of the test and if the upstream deletes a file it would be 404